package.json

```diff
@@ -17,13 +17,16 @@
   },
   "homepage": "https://github.com/owncloud/contacts#readme",
   "scripts": {
-    "test": "karma start karma.conf.js && codecov"
```

Comment:
Codecov is now run after a successful build https://travis-ci.org/owncloud/contacts/jobs/118794639#L2059 on travis

It makes some weird assumptions the first time around, so I removed it. Should work now